docs/code/StatisticsBackendTests.cpp

```diff
@@ -227,6 +227,32 @@ void get_data_examples()
     }
 }
 
+void get_data_supported_entity_kinds_examples()
+{
+    {
+        EntityId host1_id;
+        EntityId host2_id;
+
+        //CONF-GET-FASTDDS-LATENCY-SUPPORTED-ENTITY-KINDS
+        /* Get all the EntityKind pairs related to DISCOVERED_ENTITY. */
+        auto types_list = StatisticsBackend::get_data_supported_entity_kinds(DataKind::DISCOVERY_TIME);
```

Comment:
I'd not use `auto`. I think a explicit type is more visual here, since this is the actual documented function